src/components/Hero/HomeHero/index.tsx

```diff
@@ -17,6 +17,7 @@ const HomeHero = ({ heroImg }: HomeHeroProps) => {
         <Image
           src={heroImg}
           alt={t("page-index:page-index-hero-image-alt")}
+          sizes="100vw"
```

Comment:
The Next.js docs suggest adding the `style` prop with `{width: "100vw", height: "auto"}` for responsive images... Will this be handled appropriately with just `w="full" h="full"`?

Reply:
Well, I've tested it and worked well. Those are the style props we are setting in prod. Not sure if the Next suggestion would have a different outcome tbh. At the end, they are css rules, and I'd prefer to use as much as we can the Chakra style props if possible. LMK if you have any concerns about using this approach.